packages/server/src/routes/members/index.ts

```diff
@@ -126,14 +126,17 @@ async function movePrivateVideoToPublicVideo(
   }
 
   await (removeOriginal
-    ? Promise.all([
-        privateVideoRef.move(publicVideoRef),
-        privateVideoThumbnailRef.move(publicThumbnailRef)
-      ])
-    : Promise.all([
-        privateVideoRef.copy(publicVideoRef),
-        privateVideoThumbnailRef.copy(publicThumbnailRef)
-      ]));
+    ? Promise.all([privateVideoRef.move(publicVideoRef)])
```

Comment:
These don't need to be wrapped with `Promise.all` anymore since there's just one promise to await on. Same goes for all of the below

Reply:
given that this PR is going to be reverted, I was being lazy, but yes.